packages/components/app/styles/components/dropdown.scss

```
@@ -0,0 +1,410 @@
+  font-weight: var(--token-typography-font-weight-semibold);
+  line-height: var(--token-typography-body-100-line-height);
+  padding: 0.625rem 1rem 0.25rem;
+}
+
+.hds-dropdown-list-item--generic {
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.hds-dropdown-list-item--description {
+  color: var(--token-color-foreground-faint);
+  font-family: var(--token-typography-body-100-font-family);
+  font-size: var(--token-typography-body-100-font-size);
+  font-weight: var(--token-typography-font-weight-regular);
+  line-height: var(--token-typography-body-100-line-height);
+  padding: 0.125rem 1rem 0.25rem;
+}
+
+.hds-dropdown-list-item--separator {
+  position: relative;
+  height: 0.25rem;
+  width: 100%;
+
+  &::before {
+    position: absolute;
+    right: 6px;
+    left: 6px;
+    bottom: 0;
+    border-bottom: 1px solid var(--token-color-border-primary);
+    content: '';
+  }
+}
+
+.hds-dropdown-list-item__copy-item-title {
+  color: var(--token-color-foreground-faint);
+  font-family: var(--token-typography-body-100-font-family);
+  font-size: var(--token-typography-body-100-font-size); // 13
+  font-weight: var(--token-typography-font-weight-semibold);
+  line-height: var(--token-typography-body-100-line-height); // 18
+  padding: 0.125rem 0 0.25rem;
+}
+
+.hds-dropdown-list-item--copy-item {
+  box-sizing: border-box;
+  padding: 0.625rem 1rem 0.75rem;
+  width: 100%;
+
+  button {
+    @include hds-focus-ring-basic();
+    
+    background-color: transparent;
+    border-radius: 5px;
+    border: 1px solid var(--token-color-border-primary);
+    box-sizing: border-box;
+    color: var(--token-color-foreground-primary);  
+    display: flex;
+    font-family: var(--token-typography-font-stack-code);
+    justify-content: space-between;
+    padding: 0.75rem 0.5rem;
+    width: 100%;
+    
+    &:hover,
+    &.is-hover {
+      background-color: var(--token-color-surface-interactive-hover);
+      cursor: pointer;
+    }
+
+    &:focus,
+    &.is-focus {
+      //TODO this focus is just way too complex 
+      background-color:  var(--token-color-surface-action);
+      border-color: var(--token-color-focus-action-internal);
+    }
+    
+    &:active, 
+    &.is-active {
+      background-color: var(--token-color-surface-interactive-active);
+    }
+
+    &.is-success {
+      border-color: var(--token-color-border-success);
+      background-color: var(--token-color-surface-success);
+      
+      .hds-dropdown-list-item__copy-item-icon {
+        color: var(--token-color-foreground-success);
+      }
+    } 
+
+    .hds-dropdown-list-item__copy-item-text {
+      font-size: var(--token-typography-code-100-font-size);
+      font-weight: var(--token-typography-font-weight-regular);
+      line-height: var(--token-typography-code-100-line-height);
+      // max-width: 250px; // TODO we should be able to figure out the proportions here
+      overflow: hidden;
+      text-align: left;
+      text-overflow: ellipsis;
```

still missing the `white-space: nowrap;` declaration, otherwise the text will be multiline

```suggestion
      text-overflow: ellipsis;
      white-space: nowrap;
```

(notice: if we want to support also multiline, we should expose a prop to control this behaviour)